.gitignore

```diff
@@ -2,4 +2,5 @@
 .DS_Store
 bin/configlet
 bin/configlet.exe
+elm-stuff
```

Comment:
:) Is this something that everyone will have to deal with, or could it go into `.git/info/exclude`?